### PR TITLE
Add default for boolean fields on financial_type

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Add default for boolean fiels on financial_type

Before
----------------------------------------
No default db value for boolean fields is_active and is_reserved on financial type table

After
----------------------------------------
defaults added (1 for is_active & 0 for is_reserved)

Technical Details
----------------------------------------
I'll add the upgrade separately

Comments
----------------------------------------

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
While non-blocking on this there is another question for the upgrade script - is the correct default for is_deductible 1 as per the schema or 0 as per the BAO - I suspect the latter & will need to fix that before doing the schema update @seamuslee001 @monishdeb

File: CRM/Financial/DAO/FinancialType.php [hunk 1]
```diff
@@ -192,6 +193,7 @@ public static function &fields() {
           'title' => ts('Financial Type Is Active?'),
           'description' => ts('Is this property active?'),
           'where' => 'civicrm_financial_type.is_active',
+          'default' => '1',
```
Comment (eileenmcnaughton) [inline CRM/Financial/DAO/FinancialType.php:196]: The BAO is actually defaulting to 0 at the moment - but that seems inconsistent with every other place we set a default for is_active and I propose switching to TRUE (removing the line in the BAO once this is merged)

**Comment by mattwire** [comment]
This seems sensible. Please follow up with upgrader and other changes

**Comment by eileenmcnaughton** [comment]
@mattwire do you have an opinion on is_deductible?

**Comment by eileenmcnaughton** [comment]
@mattwire @seamuslee001 this is the BAO change follow up - but it highlights 'is_deductible' & also raised another question for me https://github.com/civicrm/civicrm-core/pull/19296

**Pull request merged**